akka-actor-tests/src/test/scala/akka/dispatch/ForkJoinPoolStarvationSpec.scala

```diff
@@ -10,8 +10,8 @@ import com.typesafe.config.ConfigFactory
 
 object ForkJoinPoolStarvationSpec {
   val config = ConfigFactory.parseString("""
+      |akka.actor.serialize-messages = off
```

Comment:
This took a lot of time. Doesn't explain why the test started failing when we switched FJP but should probably be done anyway.

Reply:
Do you know why it was there in the first place?

Reply:
We have it on in pretty much all test suites for some historical reason, to catch messages that aren't serializable I guess.